internal/services/keyvault/key_vault_managed_hardware_security_module_role_assignment_resource_test.go

```diff
@@ -0,0 +1,82 @@
+package keyvault_test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/keyvault/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type KeyVaultRoleAssignmentResource struct{}
```

Comment:
should this be something like
```suggestion
type KeyVaultManagedHSMRoleAssignmentResource struct{}
```